x/ecocredit/server/core/msg_retire_test.go

```diff
@@ -240,6 +269,12 @@ func (s *retire) projectSetup() {
 	s.projectKey = pKey
 }
 
+func (s *retire) AlicesAddress(a string) {
+	addr, err := sdk.AccAddressFromBech32(a)
+	require.NoError(s.t, err)
+	s.alice = addr
+}
+
```

Comment:
Nit. I've been ordering the step functions in the same order as they appear in `Given`, `When`, and `Then`. Would be nice to keep with the same structure. At the least, the public step functions should go before private functions.